delius-int/sub-projects/new-tech.tfvars

```diff
@@ -13,33 +13,50 @@ delius_api_lower_cpu_trigger = 5
 delius_api_upper_cpu_trigger = 10
 
 # Override default casenotes Config
-casenotes_conf = {
-
-}
+casenotes_conf = {}
 
 # Override default PDF Generator Config
 pdfgenerator_conf = {}
 
 # Override default Offender API Config
 offenderapi_conf = {
-    env_oracledb_servicename = "POT1NDA_TAF"
+  env_oracledb_servicename = "TSTNDA_TAF"
+  env_jwt_public_key = "LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0KTUlJQklqQU5CZ2txaGtpRzl3MEJBUUVGQUFPQ0FROEFNSUlCQ2dLQ0FRRUF6Y3g3WWJ3MkJBV3Y3U3NFSVVHUAppeUpTSURndHFCeDE5VHdtN3VJM1RYNXpSY3JabFV4Y0VtUEpHVWd5K0QySkloVmxxbWVxd0dWMkNOT3FaQmdHCmo4ZUpHQTU5aUlUemU4ZG1SSk5JYzdsNmxESmc5RE5KVk9pTHFVbFpGRENJcXplSTYzb3E2dWhjY2c1RFBpVE4KcU9HWmM4dXBOK3c1ZFpyTnYrMkdMZ3hLMnBldE1VL0JoWWVXZjNLdllJTzE2djF1dm5GT0dPMTNIb1d1NUJ0ZApTdC9UZ2NsRmhWTEVkR3c3WGJpWUhuTlpJZGh3YU5RaVlnbVhtalpWZE15Q1BETW8xMExrVjFwM1V5MTVwTU14ClVwc2xKYU8wNlZIYXJtY3ZWYzNleHg5NlpHTjE2T2U4ZWZoeG5Rdmhzd0ZrbXlYT25sSForNDI1MnJHcHlKTG8KbHdJREFRQUIKLS0tLS1FTkQgUFVCTElDIEtFWS0tLS0tCg=="
+  env_oauth2_jwt_jwk_set_uri = "https://sign-in-dev.hmpps.service.justice.gov.uk/auth/.well-known/jwks.json"
+  memory = "2048"
+  env_features_noms_update_custody = "true"
+  env_features_noms_update_booking_number = "true"
+  env_features_noms_update_keydates = "true"
+  env_features_noms_update_noms_number = "true"
+  env_smoke_test_aware = "true"
+  enable_public_lb = true
 }
 
 # Override default Elasticsearch Config
 search_conf = {}
 
 # Override default Web Frontend Config
 web_conf = {
-  image_version = "0.2.2"
+  env_offender_search_provider           = "probation-offender-search"
+  env_probation_search_base_url          = "https://probation-offender-search-dev.hmpps.service.justice.gov.uk/"
+  env_hmpps_auth_base_url                = "https://sign-in-dev.hmpps.service.justice.gov.uk/"
+  env_nomis_api_base_url                 = "https://api-dev.prison.service.justice.gov.uk/"
 }
 
-
 # Override default Offender Poll Push Config
-offenderpollpush_conf = {}
+offenderpollpush_conf = {
+  env_sns_arn_topic = "arn:aws:sns:eu-west-2:754256621582:cloud-platform-Digital-Prison-Services-453cac1179377186788c5fcd12525870"
```

Comment:
You can remove these since offenderpollpush_conf is no longer used (it was moved to cloud platform). Probably needs clearing up from test etc as

Reply:
Have removed the values - can the config block be removed completely?